tests/templates/test_embeddings/test_amplitude.py

```diff
@@ -308,6 +326,23 @@ def test_tf(self, tol):
 
         assert qml.math.allclose(res, res2, atol=tol, rtol=0)
 
+    def test_tf_jit(self, tol):
+        """Tests tf tensors when using JIT."""
+
+        tf = pytest.importorskip("tensorflow")
+
+        features = tf.Variable([1 / 2, 0, 1 / 2, 0, 1 / 2, 1 / 2, 0, 0])
+
+        dev = qml.device("default.qubit", wires=3)
+
+        circuit = tf.function(jit_compile=True)(qml.QNode(circuit_template, dev, interface="tf"))
+        circuit2 = tf.function(jit_compile=True)(qml.QNode(circuit_decomposed, dev, interface="tf"))
```

Comment:
Minor: would we like to parametrize `jit_compile` here too?

Reply:
I think it should be okay, since the behavior of `is_abstract` is already tested in both cases!